scilpy/reconst/dti.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+supported_tensor_formats = ['fsl', 'nifti', 'mrtrix', 'dipy']
+tensor_format_description = \
+    """
+    Dipy's order is [Dxx, Dxy, Dyy, Dxz, Dyz, Dzz]
+       Shape: [i, j , k, 6].
+       Ref: [See _lt_rows and _lt_cols in dipy.reconst.dti]
```

Comment:
Add link to documentation like the others

Reply:
It didn't exist as of yesterday ;) They merged my Pull request this morning but it will only appear after the next release, I guess. 

Do you want me to put this link already? (The next release is planned soon I think)
https://dipy.org/documentation/1.5.0/examples_built/reconst_dti/#example-reconst-dti

Else, I can refer to a line in the Github repo
https://github.com/dipy/dipy/blob/master/dipy/reconst/dti.py#L1639

Which do you prefer?

Reply:
Ahhhh. As you wish then, I don't mind.